linux/debian/mozillavpn.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
```

Comment:
Where should this file be installed? And what does install it? Is the debian package to take care of it?